### PR TITLE
Add file overviews

### PR DESCRIPTION
**Pull request by subsidel**

### CODE REVIEW

File: static/script/devices/anim/styletopleft.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs module containing base antie.devices.anim.css3.transitionendpoints class.
```
Comment (AlexChesters) [inline static/script/devices/anim/styletopleft.js]: Copy and paste error?

File: static/script/devices/anim/tween.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs module containing base antie.devices.anim.css3.transitionendpoints class.
```
Comment (AlexChesters) [inline static/script/devices/anim/tween.js]: Copy and paste error?

File: static/script/devices/broadcastsource/basetvsource.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs module containing base antie.devices.anim.css3.transitionendpoints class.
```
Comment (AlexChesters) [inline static/script/devices/broadcastsource/basetvsource.js]: Copy and paste error?

File: static/script/devices/broadcastsource/hbbtvsource.js [hunk 4]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs module containing base antie.devices.anim.css3.transitionendpoints class.
```
Comment (AlexChesters) [inline static/script/devices/broadcastsource/hbbtvsource.js]: Copy and paste error?

File: static/script/devices/broadcastsource/samsungtvsource.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 /* global webapis */
 
 /**
+ * @fileOverview Requirejs module containing base antie.devices.anim.css3.transitionendpoints class.
```
Comment (AlexChesters) [inline static/script/devices/broadcastsource/samsungtvsource.js]: Copy and paste error?

File: static/script/devices/broadcastsource/tizentvsource.js [hunk 6]
```diff
@@ -1,6 +1,7 @@
 /* global tizen */
 
 /**
+ * @fileOverview Requirejs module containing base antie.devices.anim.css3.transitionendpoints class.
```
Comment (AlexChesters) [inline static/script/devices/broadcastsource/tizentvsource.js]: Copy and paste error?

File: static/script/devices/browserdevice.js [hunk 7]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs module containing base antie.devices.anim.css3.transitionendpoints class.
```
Comment (AlexChesters) [inline static/script/devices/browserdevice.js]: Copy and paste error?

File: static/script/devices/device.js [hunk 8]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs modifier to use native JSON decoding/encoding if supported by the browser
```
Comment (AlexChesters) [inline static/script/devices/device.js]: Copy and paste error?

File: static/script/devices/exit/broadcast/netcast.js [hunk 9]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs modifier to use native JSON decoding/encoding if supported by the browser
```
Comment (AlexChesters) [inline static/script/devices/exit/broadcast/netcast.js]: Copy and paste error?

File: static/script/devices/exit/broadcast/samsung_maple.js [hunk 10]
```diff
@@ -1,6 +1,7 @@
 /* global Common */
 
 /**
+ * @fileOverview Requirejs modifier to use native JSON decoding/encoding if supported by the browser
```
Comment (AlexChesters) [inline static/script/devices/exit/broadcast/samsung_maple.js]: Copy and paste error?

File: static/script/devices/exit/closewindow.js [hunk 11]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs modifier to use native JSON decoding/encoding if supported by the browser
```
Comment (AlexChesters) [inline static/script/devices/exit/closewindow.js]: Copy and paste error?

File: static/script/devices/exit/netcast.js [hunk 12]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs module containing base antie.devices.exit.history class
```
Comment (AlexChesters) [inline static/script/devices/exit/netcast.js]: Copy and paste error?

File: static/script/devices/googletv.js [hunk 13]
```diff
@@ -1,4 +1,6 @@
 /**
+ * @fileOverview Requirejs module containing the antie.devices.exit.tivo class for use with
+ * tivo html-based system browser.
```
Comment (AlexChesters) [inline static/script/devices/googletv.js]: Copy and paste error?

File: static/script/events/keyevent.js [hunk 14]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs module containing the antie.events.FocusEvent class.
```
Comment (AlexChesters) [inline static/script/events/keyevent.js]: Copy and paste error?

File: static/script/events/mediaevent.js [hunk 15]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs module containing the antie.events.MediaErrorEvent class.
```
Comment (AlexChesters) [inline static/script/events/mediaevent.js]: Copy and paste error?

File: static/script/events/mediasourceerrorevent.js [hunk 16]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs module containing the antie.events.MediaErrorEvent class.
```
Comment (AlexChesters) [inline static/script/events/mediasourceerrorevent.js]: Copy and paste error?

File: static/script/historian.js [hunk 17]
```diff
@@ -1,4 +1,5 @@
 /**
+ * @fileOverview Requirejs module containing base antie.Formatter class.
```
Comment (AlexChesters) [inline static/script/historian.js]: Copy and paste error?

**Comment by subsidel** [comment]
Thanks, Alex :)

**Pull request merged**